notification-portlet-webapp/src/main/webapp/WEB-INF/jsp/show-alerts.jsp

```diff
@@ -31,7 +31,7 @@
 <c:if test="${!usePortalJsLibs}">
     <rs:aggregatedResources path="/jQueryUIResources.xml"/>
 </c:if>
-<script src="<c:url value="/scripts/jquery.notice.min.js"/>" type="text/javascript"></script>
+<rs:aggregatedResources path="/simpleListLocalResources.xml"/>
```

Comment:
I suggest reverting this line.  show-alerts doesn't need styles/simple-list.css which is in the resources file.